### PR TITLE
Add release notes for v19.2.0-rc.1

### LINKED ISSUE #5611: Release notes for v19.2.0-rc.1
See https://github.com/cockroachdb/cockroach/issues/41562.

### PR DESCRIPTION
**Pull request by jseldess**
Fixes #5611.

### CODE REVIEW

**Comment by jseldess** [comment]
___
*[releases/v19.2.0-rc.1.md, line 41 at r1](https://reviewable.io/reviews/cockroachdb/docs/5633#-LrMh_Dl6b1DWElPdHvx:-LrMh_Dl6b1DWElPdHvy:b-qdmfxz) ([raw file](https://github.com/cockroachdb/docs/blob/267c92e200f07844994a723b0924df67925ffcb8/releases/v19.2.0-rc.1.md#L41)):*
> ```Markdown
> ### SQL language changes
> 
> - `INTERVAL(6)` is now supported as an alias for [`INTERVAL`](../v19.2/interval.html), for compatibility with third-party tools. [#41447][#41447]
> ```

@jordanlewis, are you ok with the way I've rewritten these sql change release notes?

**Comment by jseldess** [comment]
@jordanlewis, please review the third commit, too.

File: _includes/v19.2/faq/sql-query-logging.md [hunk 1]
```diff
@@ -32,12 +32,16 @@ With this setting on, each node of the cluster writes all SQL queries it execute
...
-Using the CLI to start a new node, pass the `--vmodule` flag to the [`cockroach start`](start-a-node.html) command. For example, to start a single node locally and log all SQL queries it executes, you'd run:
+Using the CLI to start a new node, pass the `--vmodule` flag to the [`cockroach start`](start-a-node.html) command. For example, to start a single node locally and log all client-generated SQL queries it executes, you'd run:
 
 ~~~ shell
-$ cockroach start --insecure --listen-addr=localhost --vmodule=exec_log=2
+$ cockroach start --insecure --listen-addr=localhost --vmodule=exec_log=2 --join=<join addresses>
 ~~~
 
+{{site.data.alerts.callout_success}}
+To log CockroachDB-generated SQL queries as well, use `exec_logs=3`.
```
Comment (jordanlewis) [inline _includes/v19.2/faq/sql-query-logging.md]: You probably want `--vmodule=exec_log=3` here, not `exec_logs=3`.

File: releases/v19.2.0-rc.1.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v19.2.0-rc.1
+~~~
+
+### SQL language changes
+
+- `INTERVAL(6)` is now supported as an alias for [`INTERVAL`](../v19.2/interval.html), for compatibility with third-party tools. [#41447][#41447]
+- Added the `pg_available_extensions` table, for compatibility with third-party tools. [#41447][#41447]
+- The `pg_attdef()` and `pg_get_indexdef()` [built-in functions](../v19.2/functions-and-operators.html) no longer include type annotations on default expressions, and `pg_get_indexdef()` now includes the index method in the returned `CREATE INDEX` statement, for compatibility with third-party tools. [#41447][#41447]
```
Comment (jordanlewis) [inline releases/v19.2.0-rc.1.md]: This should say `attrdef` not `attdef`.

File: releases/v19.2.0-rc.1.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+- Clarified epoch-based vs. expiration-based leases in the [Replication Layer](../v19.2/architecture/replication-layer.html#epoch-based-leases-table-data) architecture documentation. [#5578][#5578]
+- Updated the list of [Enterprise features](../v19.2/enterprise-licensing.html). [#5599][#5599]
+- Documented the new [`--geo-partitioned-replicas` flag of the `cockroach demo` command](../v19.2/cockroach-demo.html#run-cockroach-demo-with-geo-partitioned-replicas), which starts a 9-node in-memory cluster with the [Geo-Partitioned Replicas](../v19.2/topology-geo-partitioned-replicas.html) data topology applied to the `movr` database. This lets you quickly simulate one of the most effective data topology patterns for getting low latency in a multi-region cluster. [#5587][#5587]  
+
+### Contributors
+
+This release includes 12 merged PRs by 10 authors. We would like to thank the following contributor from the CockroachDB community:
+
+- Aayush Shah (first-time contributor)
```
Comment (jordanlewis) [inline releases/v19.2.0-rc.1.md]: Aayush is an intern. Do we normally include employees on this list?

File: releases/v19.2.0-rc.1.md (hunk 2 above)
Comment (jseldess) [reply releases/v19.2.0-rc.1.md]: Done.

File: releases/v19.2.0-rc.1.md (hunk 3 above)
Comment (jseldess) [reply releases/v19.2.0-rc.1.md]: Nope. Removed. Thanks.

Comment (jseldess) [reply releases/v19.2.0-rc.1.md]: Nope. Removed. Thanks.

File: releases/v19.2.0-rc.1.md (hunk 2 above)
Comment (jseldess) [reply releases/v19.2.0-rc.1.md]: Done.

File: _includes/v19.2/faq/sql-query-logging.md (hunk 1 above)
Comment (jseldess) [reply _includes/v19.2/faq/sql-query-logging.md]: Done.

**Pull request merged**